vm-setup/requirements.yml

```diff
@@ -1,3 +1,3 @@
-- src: https://github.com/metal3-io/ansible-role-golang.git
-  verison: master
+- src: https://github.com/fubarhouse/ansible-role-golang/ansible-role-golang.git
```

Comment:
Shouldn't this be https://github.com/fubarhouse/ansible-role-golang.git?

Reply:
fixed